modules/setting/lfs.go

```diff
@@ -20,25 +20,34 @@ var LFS = struct {
 	MaxFileSize     int64         `ini:"LFS_MAX_FILE_SIZE"`
 	LocksPagingNum  int           `ini:"LFS_LOCKS_PAGING_NUM"`
 
-	Storage
+	Storage *Storage
 }{}
 
-func loadLFSFrom(rootCfg ConfigProvider) {
+func loadLFSFrom(rootCfg ConfigProvider) error {
 	sec := rootCfg.Section("server")
 	if err := sec.MapTo(&LFS); err != nil {
-		log.Fatal("Failed to map LFS settings: %v", err)
+		return fmt.Errorf("failed to map LFS settings: %v", err)
 	}
 
-	lfsSec := rootCfg.Section("lfs")
-	storageType := lfsSec.Key("STORAGE_TYPE").MustString("")
+	lfsSec, _ := rootCfg.GetSection("lfs")
 
 	// Specifically default PATH to LFS_CONTENT_PATH
 	// DEPRECATED should not be removed because users maybe upgrade from lower version to the latest version
 	// if these are removed, the warning will not be shown
 	deprecatedSetting(rootCfg, "server", "LFS_CONTENT_PATH", "lfs", "PATH", "v1.19.0")
-	lfsSec.Key("PATH").MustString(sec.Key("LFS_CONTENT_PATH").String())
-
-	LFS.Storage = getStorage(rootCfg, "lfs", storageType, lfsSec)
+	lfsContentPath := sec.Key("LFS_CONTENT_PATH").String()
+	if lfsContentPath != "" {
+		LFS.Storage = &Storage{
+			Type: LocalStorageType,
+			Path: lfsContentPath,
+		}
```

Comment:
I think the old logic is better, use unified  `getStorage` to create the storage. Manually constructing `&Storage{` seems fragile.

And, to be simple, the `LFS_CONTENT_PATH` could be simply removed (report a fatal error to make sure users finishes their ini config migration), then no fallback logic is needed.

Reply:
LFS_CONTENT_PATH Removed